lib/gui/modules/error.js

```diff
@@ -41,6 +41,10 @@ error.service('ErrorService', function(AnalyticsService, OSDialogService) {
    * ErrorService.reportException(new Error('Something happened'));
    */
   this.reportException = (exception) => {
+    if (angular.isUndefined(exception)) {
```

Comment:
This is really minor, but I guess we should use Lodash where possible instead of Angular functions given that we will switch from it in the near future.